syft/serde/serde.py

```diff
@@ -123,6 +123,14 @@
 
 ## SECTION: High Level Simplification Router
 def _force_full_simplify(obj: object) -> object:
+    """To force a full simplify genrally if the usual _simplify is not suitable.
+    
+    Args:
+        The simplified object
```

Comment:
I was saying the simpified object is the return not the arg

Reply:
Sorry